app-admin/prelude-manager/prelude-manager-3.0.0.ebuild

```diff
@@ -0,0 +1,68 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+
+inherit autotools eutils systemd
+
+DESCRIPTION="Bus communication for all Prelude modules"
+HOMEPAGE="https://www.prelude-siem.org"
+SRC_URI="https://www.prelude-siem.org/pkg/src/3.0.0/${PF}.tar.gz"
+
+LICENSE="GPL-2+"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="tcpwrapper xml geoip dbx systemd"
+
+RDEPEND="net-libs/gnutls
+	dbx? ( dev-libs/libpreludedb )
+	tcpwrapper? ( sys-apps/tcp-wrappers )
+	xml? ( dev-libs/libxml2 )
+	geoip? ( dev-libs/libmaxminddb )"
+
+DEPEND="${RDEPEND}
+	virtual/pkgconfig"
+
+PATCHES=(
+	"${FILESDIR}/${P}-conf.patch"
+	"${FILESDIR}/${P}-configure.patch"
+	"${FILESDIR}/${P}-run.patch"
+)
+
+src_prepare() {
+	default_src_prepare
+
+	mv "${WORKDIR}/${PF}/configure.in" "${WORKDIR}/${PF}/configure.ac" || die "mv failed"
+
+	eautoreconf
+}
+
+src_configure() {
+	econf \
+		--localstatedir=/var \
+		$(use_enable dbx libpreludedb) \
+		$(use_with tcpwrapper libwrap) \
+		$(use_enable xml xmlmod) \
+		$(use_enable geoip libmaxminddb)
+}
+
+src_install() {
+	default_src_install
+
+	rm -rf "${D}/var/run/${PN}" || die "rm failed"
+
+	prune_libtool_files --modules
+
+	if use systemd ; then
```

Comment:
Note : note for prewikka because user should use Apache as it is said in https://www.prelude-siem.org/projects/prelude/wiki/InstallingPreludePrewikka#Run-Prewikka